packages/node_modules/@ciscospark/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -122,21 +122,60 @@ export const transforms = toArray('inbound', {
   },
 
   /**
-   * Decrypts a content field which contains files
+   * Decrypts a content field
    * @param   {Object} ctx            An object containing a spark instance and transform prop
    * @param   {String} key            KMS key
    * @param   {Object} content        An object with properties to be decrypted
    * @parma   {Array}  content.files  An array of files to decrypt by calling decryptObject
    * @returns {[type]}                [description]
    */
   decryptContent(ctx, key, content) {
+    if (content.contentCategory === 'links') {
+      return ctx.transform('decryptContentLinks', key, content);
+    }
+
+    return ctx.transform('decryptContentFiles', key, content);
+  },
+
+  /**
+   * Decrypts a content field which contains files and possibly links
+   * @param   {Object} ctx            An object containing a spark instance and transform prop
+   * @param   {String} key            KMS key
+   * @param   {Object} content        An object with properties to be decrypted
+   * @parma   {Array}  content.files  An array of files to decrypt by calling decryptObject
+   * @returns {[type]}                [description]
+   */
+  decryptContentFiles(ctx, key, content) {
     if (!content.files || !content.files.items || !Array.isArray(content.files.items)) {
       return Promise.resolve();
     }
 
     const promises = content.files.items.map((item) => ctx.transform('decryptObject', key, item));
     promises.push(ctx.transform('decryptComment', key, content));
 
+    if (content.links && content.links.items && Array.isArray(content.links.items)) {
```

Comment:
Is this a different type of `link`? I would expect this to call `decryptContentLinks` instead of `decryptObject`.

Reply:
Calling decryptContentLinks here would decrypt the comment twice. If you want, I can refactor the mapping call to links.items that decrypts each one but it is only one line.

Reply:
No, it's fine. Thank you for clarifying.